packages/core/src/Stores/traders-hub-store.js

```diff
@@ -381,13 +381,22 @@ export default class TradersHubStore extends BaseStore {
     }
 
     getAvailableMt5Accounts() {
+        const { gaming_company } = this.root_store.client.landing_companies;
+
+        const only_gaming_company = gaming_company?.shortcode === 'svg';
```

Comment:
the `financial_company: none` should be added as a condition as well.
and please define this as a `computed` out of the function scope for the further usage.